src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -286,11 +285,7 @@ private async Task<IEnumerable<string>> GetPackageDownloadUrlsAsync(IEnumerable<
                 {
                     var installer = _workloadInstaller.GetPackInstaller();
 
-                    var packUrls = workloadIds
-                        .SelectMany(workloadId => _workloadResolver.GetPacksInWorkload(workloadId.ToString()))
-                        .Distinct()
-                        .Select(packId => _workloadResolver.TryGetPackInfo(packId))
-                        .Where(pack => pack != null)
+                    var packUrls = GetPacksToInstall(workloadIds)
```

Comment:
Is this variable name misleading? It doesn't seem like GetPacksToInstall is returning URLs or is there another overload not in the PR?